packages/graphql-eslint-rules/src/__tests__/fragment-naming-conventions.test.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Taken from https://github.com/dotansimha/graphql-eslint/blob/300f73be802bdd06432a5df34939521d1ce0d93b/packages/plugin/tests/require-id-when-available.spec.ts
+ * MIT license https://github.com/dotansimha/graphql-eslint/blob/300f73be802bdd06432a5df34939521d1ce0d93b/LICENSE
+ */
```

Comment:
I don't think this is still needed for what you have below. The code below is basically an entire new test case that happens to use the same rule tester API.